third_party/2and3/pynamodb/attributes.pyi

```diff
@@ -39,8 +39,8 @@ class UnicodeSetAttribute(SetMixin, Attribute[Set[Text]]):
 class UnicodeAttribute(Attribute[Text]):
     def __get__(self, instance: Any, owner: Any) -> Text: ...
 
-class JSONAttribute(Attribute[Dict[Text, Any]]):
-    def __get__(self, instance: Any, owner: Any) -> Dict[Text, Any]: ...
+class JSONAttribute(Attribute[Union[Dict[Text, Any], List[Any]]]):
```

Comment:
This is technically not correct based on https://github.com/antlr/grammars-v4/blob/75c2026b1f207c80334c7fb0043da81998feda50/json/JSON.g4#L25-L33